contracts/TracerPerpetualSwaps.sol

```diff
@@ -198,17 +202,17 @@ contract TracerPerpetualSwaps is ITracerPerpetualSwaps, SimpleDex, Ownable, Safe
 
         int256 baseChange = (fillAmount.toInt256().mul(order.price)).div(priceMultiplier.toInt256());
         require(baseChange > 0, "TCR: Base change <= 0");
+        
+        // Settle accounts
```

Comment:
I assume the moving of this is just so when we update leverage it gets the most up to date values?

Reply:
Yeah you and I discussed it IRL last week and agreed this was the way to go.